kuma/users/basket.py

```diff
@@ -0,0 +1,18 @@
+import requests
+from celery import task
+
+BASKET_SUBSCRIPTION_URL = 'https://basket.mozilla.org/news/subscribe/'
+
+
+@task
+def subscribe(user):
```

Comment:
I would recommend:
```suggestion
def subscribe(email, username, locale):
```
instead, in order to avoid the serialization/deserialization (i.e., pickling/unpickling) of the user instance when inserted-into/removed-from the Celery task queue, and since the three pieces of information needed are all strings.

Reply:
Makes sense! No need to waste cycles on uneeded data, will fix. Thanks!